docs/guides/external-packages.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: How to use external packages
+description: "A guide on configuring your truss to use external packages"
+---
+
+You might encounter a situation where you have to incorporate a third-party package into your truss. Truss has a few different mechanisms to support this.
```

Comment:
Not 100% sure what third-party package means here, to me that suggests packages from PyPi not your own code.

Also instead of saying a few different mechasisms could you enumerate them up top, maybe in a bulleted list?